docs/getting-started/configure-app-with-plugins.md

```diff
@@ -3,4 +3,31 @@ id: configure-app-with-plugins
 title: Configuring App with plugins
 ---
 
+## Adding existing plugins to your app
+
 Coming soon!
+
+### Adding a plugin page to the Sidebar
+
+In a standard Backstage app created with `@backstage/create-app`, the sidebar is
```

Comment:
> app created with `@backstage/create-app`,

Maybe make this a link?

Reply:
Linked to folder in this repo, or were you thinking npm?

Reply:
The doc that describes this process of creating an app.

Reply:
Ah, wasn't in my mind since this page is directly following that. Makes sense either way